.github/workflows/autoformat.yml

```diff
@@ -0,0 +1,71 @@
+name: Autoformat
+on: issue_comment
+
+defaults:
+  run:
+    shell: bash --noprofile --norc -exo pipefail {0}
+
+jobs:
+  # Commenting '/autoformat' on a pull request triggers this workflow.
+  # This allows both maintainers and contributors to trigger this workflow.
+  parse-comment:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }}
+    outputs:
+      matched: ${{ steps.parse-comment.outputs.matched }}
+    steps:
+      - name: Parse comment
+        id: parse-comment
+        run: |
+          # Trim whitespaces and newlines
+          trimmed=$(echo "${{ github.event.comment.body }}" | tr -d '[:space:]')
+          matched=$([[ "$trimmed" == "/autoformat" ]] && echo "true" || echo "false")
+          echo "::set-output name=matched::$matched"
+  autoformat:
+    runs-on: ubuntu-latest
+    needs: parse-comment
+    if: ${{ needs.parse-comment.outputs.matched == 'true' }}
+    steps:
+      - uses: actions/github-script@v4
+        id: get-pr
+        with:
+          script: |
+            const {
+              repo: { owner, repo },
+              issue: { number: pull_number },
+            } = context;
+            const pr = await github.pulls.get({ owner, repo, pull_number });
+            return pr.data
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ fromJSON(steps.get-pr.outputs.result).head.repo.full_name }}
+          ref: ${{ fromJSON(steps.get-pr.outputs.result).head.ref }}
+          # As mentioned in the doc below, if we use `secrets.GITHUB_TOKEN`, a commit cerated by
+          # this workflow will not trigger other GitHub Actions workflows:
+          # https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow
+          # To make it work, register a personal access token for the mlflow-automation bot as
```

Comment:
Should we add these steps to the CONTRIBUTING guidelines?

Reply:
For someone who wants to run this workflow on their fork?

Reply:
I'm not sure if there are any regular contributors who do this? I suppose if they are they're going to be reading over the Actions yaml configs when they set it up though.